docs/pages/eas-update/debug-updates.mdx

```diff
@@ -263,6 +263,74 @@ It may be helpful to see which assets are included in our update bundle. We can
 
 <Terminal cmd={['$ npx expo export']} />
 
+### Debugging of native code in an app enabled for updates
+
+Normally, if we have a project with `expo-updates` installed, we need to make a release build for `expo-updates` to be enabled (debug builds behave like normal RN project debug builds).
+
+To make it easier to test and debug native code, we can follow these steps to create a debug build of the app with expo-updates enabled.
+
+#### iOS local builds
+
+- Set the environment variable
+```sh
+export EX_UPDATES_NATIVE_DEBUG=1
+```
+- Reinstall pods with `pod install` or `npx pod-install`. The `expo-updates` podspec now detects this environment variable, and makes changes so that the debug code that would normally load from the Metro packager is bypassed, and the app is built with the EXUpdates bundle and other dependencies needed to load updates from EAS.
+- Execute a debug build of the app with Xcode or from the command line.
+
+#### Android local builds
+
+- As above in the iOS build, set the environment variable
+```sh
+export EX_UPDATES_NATIVE_DEBUG=1
+```
+- Modify your app's `AndroidManifest.xml` to select a channel for which this app will receive updates, as in the example below. (In an EAS build, this value is set automatically to the channel for the EAS profile being built.)
```

Comment:
```suggestion
- Modify your app's **AndroidManifest.xml** to select a channel for which this app will receive updates, as in the example below. (In an EAS build, this value is set automatically to the channel for the EAS profile being built.)
```

Filenames are bold as [per our writing styleguide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#filenames-as-bold-text) for consistency.